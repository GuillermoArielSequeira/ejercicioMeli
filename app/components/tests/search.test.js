const React = require('react');
const { shallow } = require('enzyme');
const Search = require('../search');

describe('search Component', () => {
  it('render search correctly', () => {
    const wrapper = shallow(<Search />);
    expect(wrapper).toMatchSnapshot();
  })
  it('simulate onChange', () => {

    const event = {
      preventDefault: () => { },
      target: { value: 'the-value' }
    };

    const wrapper = shallow(<Search />);
    wrapper.find('input').at(0).simulate('change', event);
    expect(wrapper.find('input').at(1).prop('value')).toBe('the-value');

  })
});
