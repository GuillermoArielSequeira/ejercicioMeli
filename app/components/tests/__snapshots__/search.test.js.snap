// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search Component render search correctly 1`] = `ShallowWrapper {}`;
